playground/mocks/data/idp/idx/error-authenticator-reset-password-requirement.json

```diff
@@ -0,0 +1,211 @@
+{
+  "version":"1.0.0",
+  "stateHandle":"02hZGbwbPSfpkxvQJACvqXcLyhUYfnGwM2k6tpCJX5",
+  "expiresAt":"2021-05-18T21:08:24.000Z",
+  "intent":"LOGIN",
+  "remediation":{
+    "type":"array",
+    "value":[
+      {
+        "rel":[
+          "create-form"
+        ],
+        "name":"reset-authenticator",
+        "relatesTo":[
+          "$.currentAuthenticator"
+        ],
+        "href":"http://localhost:3000/idp/idx/challenge/answer",
+        "method":"POST",
+        "produces":"application/ion+json; okta-version=1.0.0",
+        "value":[
+          {
+            "name":"credentials",
+            "type":"object",
+            "form":{
+              "value":[
+                {
+                  "name":"passcode",
+                  "label":"New password",
+                  "secret":true,
+                  "messages":{
+                    "type":"array",
+                    "value":[
+                      {
+                        "message":"Password requirements were not met. Password requirements: at least 8 characters, a lowercase letter, an uppercase letter, a number, no parts of your username. Your password cannot be any of your last 4 passwords.",
+                        "i18n":{
+                          "key":"password.passwordRequirementsNotMet",
```

Comment:
Do we have `password.passwordRequirementsNotMet` in the login bundle ? or am I missing something ? I see we have `registration.error.password.passwordRequirementsNotMet`

Reply:
Ah nvm I see the key used in AuthenticatorUtils

Reply:
No we don't have that. But we do have `registration.error.password.passwordRequirementsNotMet` and I'm using it in `AuthenticatorUtil`. i18nTransformer wouldn't work because of the params. We don't want to show the params so I'm just modifying `AuthenticatorUtil` to show `registration.error.password.passwordRequirementsNotMet` instead.

Reply:
Just so I understand, we are not going to use the params here, but the API still sends it ? Ideally there would not be any params and we could add this to i18nTransformer right ?

Reply:
yes, that's correct